gcp_airflow_foundations/operators/gcp/hds/load_hds_taskgroup.py

```diff
@@ -61,6 +63,16 @@ def hds_builder(
             "Invalid HDS table type", hds_table_config.hds_table_type
         )
 
+    create_dataset = CustomBigQueryCreateEmptyDatasetOperator(
```

Comment:
Could maybe optimize to run just once, instead of every time. Can add an improvement jira 

https://medium.com/@plieningerweb/use-apache-airflow-to-run-task-exactly-once-6fb70ca5e7ec

Reply:
Will add it on Jira too look into that.